_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -52,6 +52,7 @@ class HealthCareApp extends React.Component {
     this.context.store.dispatch(ensureFieldsInitialized(path));
     if (validations.isValidSection(path, sectionData)) {
       hashHistory.push(this.getUrl('next'));
+      sectionData.sectionComplete = true;
```

Comment:
@awong-dev Could use your input here, I'm wondering if we want to dispatch an action here since we're updating the store, instead of just changing it directly?